README.md

```diff
@@ -63,18 +63,18 @@ From `openapi-dev-tool` command-line interface, you can:
 
 - **Serve one or several OpenAPI files** specified in [configuration file](#configuration-file) and written in YAML or JSON
 - **Live reloading** to view dynamically your changes in your browser in real-time
-- **Bundle OpenAPI files** to merge the whole of files before serving. By using [OpenAPI remote reference](https://swagger.io/docs/specification/using-ref/), you can work on several files. That facilitates the maintainability and avoid having a large OpenAPI file!
+- **Bundle OpenAPI files** to merge all the files before serving. By using [OpenAPI remote reference](https://swagger.io/docs/specification/using-ref/), you can work on several files. It improves the maintainability and avoids having a large OpenAPI file!
 - **Several documentation viewers embedded** with the possibility to [override the templates](#templates-override) used:
   - Redoc (http://localhost:3000)
   - RedocPro (http://localhost:3000) --> RedocPro to have "try it" feature
```

Comment:
What is the meaning of `RedocPro to have "try it" feature`?

Reply:
RedocPro comes with a "try it" feature which doen't exist in Redoc